tests/core/basic20/task.yaml

```diff
@@ -1,10 +1,20 @@
 summary: Check basic core20 system functionality
 
-systems: [ubuntu-core-20-*]
+systems:
+  - ubuntu-core-20-*
+  - ubuntu-core-22-*
```

Comment:
shouldn't new tests for core22 be put under a basic22/ folder instead of being part of the basic20 folder?

Reply:
Would prefer to rename it `basic20+` since there should not be much changes there.